six/common/builtins/aggregator.c

```diff
@@ -222,25 +222,27 @@ static PyObject *submit_event(PyObject *self, PyObject *args)
     PyObject *py_tags = NULL; // borrowed
     char *check_id = NULL;
     event_t *ev = NULL;
+    PyObject * retval = NULL;
 
     // aggregator.submit_event(self, check_id, event)
     if (!PyArg_ParseTuple(args, "OsO", &check, &check_id, &event_dict)) {
         PyGILState_Release(gstate);
         // error is set by PyArg_ParseTuple but we return NULL to raise
-        return NULL;
+        retval = NULL;
+        goto gstate_cleanup;
```

Comment:
since we jump to `gstate_cleanup` we should remove the `PyGILState_Release` in 229

Reply:
oops!